apiserver/plane/bgtasks/issue_activites_task.py

```diff
@@ -1428,12 +1526,58 @@ def issue_activity(
                     )
 
             # Bulk create notifications
-            Notification.objects.bulk_create(bulk_notifications, batch_size=100)
+            Notification.objects.bulk_create(
+                bulk_notifications, batch_size=100)
+
+            # Add Mentioned as Issue Subscribers
+            IssueSubscriber.objects.bulk_create(mention_subscribers, batch_size=100)
+            
+            mention_notifications = []
+
+            # Send Notifications to the Mentioned and Add Mentioned as Subscribers
+            for mention_id in new_mentions:
+                for issue_activity in issue_activities_created:
+                    if (issue_activity.verb == "created" or issue_activity.verb == "updated"):
+                        mention_notifications.append(
+                            Notification(
+                                workspace=project.workspace,
+                                sender="in_app:issue_activities",
+                                triggered_by_id=actor_id,
+                                receiver_id=mention_id,
+                                entity_identifier=issue_id,
+                                entity_name="issue",
+                                project=project,
+                                message=f"You have been mentioned in the issue {issue.name}",
+                                data={
+                                    "issue": {
+                                        "id": str(issue_id),
+                                        "name": str(issue.name),
+                                        "identifier": str(issue.project.identifier),
+                                        "sequence_id": issue.sequence_id,
+                                        "state_name": issue.state.name,
+                                        "state_group": issue.state.group,
+                                    },
+                                    "issue_activity": {
+                                        "id": str(issue_activity.id),
+                                        "verb": str(issue_activity.verb),
+                                        "field": "description",
+                                        "actor": str(issue_activity.actor_id),
+                                        "new_value": str(issue_activity.new_value),
+                                        "old_value": str(issue_activity.old_value),
+                                    },
+                                },
+                            )
+                        )
+
+            # Send those Bulk Notifications
+            Notification.objects.bulk_create(
+                mention_notifications, batch_size=100)
 
         return
     except Exception as e:
         # Print logs if in DEBUG mode
         if settings.DEBUG:
+            print("====================EXCEPTION OCCURED====================")
```

Comment:
@henit-chobisa please remove this line before merge

Reply:
I have removed it. Actually that like was kinda intentional, because celery serves exceptions as warning only, so I thought, it can be better to have a symbol.